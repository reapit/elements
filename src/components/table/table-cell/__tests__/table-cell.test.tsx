import { render } from '@testing-library/react'
import { TableCell } from '../index'

describe('TableCell', () => {
  test('should match snapshot', () => {
    const { asFragment } = render(<TableCell>td</TableCell>)
    expect(asFragment()).toMatchSnapshot()
  })
})
