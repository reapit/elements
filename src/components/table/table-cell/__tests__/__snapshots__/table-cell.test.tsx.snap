// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TableCell > should match snapshot 1`] = `
<DocumentFragment>
  <td
    class="mocked-styled-0 el-table-cell"
    data-alignment="left"
    role="cell"
    style="--edd0gm5-0: auto; --edd0gm5-1: auto; --edd0gm5-2: auto;"
  >
    <div
      class="mocked-styled-1 el-table-cell-content"
      data-flex-direction="column"
      style="--e1i8lsm-0: nowrap;"
    >
      <div
        class="mocked-styled-2 el-table-toolbar"
      >
        <div
          class="mocked-styled-3 el-table-toolbar-description"
        >
          125 Properties
        </div>
        <div
          class="mocked-styled-4 el-table-toolbar-actions"
        >
          Page Size menu component goes here
        </div>
      </div>
    </div>
  </td>
</DocumentFragment>
`;
